docs/sanitizers/asan-flags.md

```diff
@@ -0,0 +1,132 @@
+---
+title: "AddressSanitizer Runtime Options"
+description: "List of runtime options for the Microsoft C/C++ Address Sanitizer"
+ms.date: 8/05/2025
+helpviewer_keywords: ["AddressSanitizer options", "AddressSanitizer flags", "Address Sanitizer options", "Address Sanitizer flags", "asan options", "asan flags", "sanitizer flags", "asan_options", "AddressSanitizerFlags", "CommonSanitizerFlags"]
+---
+
+# AddressSanitizer Runtime Options
+
+The following is a list of supported runtime options that can be set by the `ASAN_OPTIONS` environment variable or by overriding the `__asan_default_options()` user function. Reference: [AddressSanitizer Runtime Options](./asan-runtime.md#runtime-options). Note that while these mirror the options available in Clang's ASan, the description will differ by varying amounts in consideration of the MSVC ASan runtime.
```

Comment:
What is the reference about? Are you wanting to say: "For more information about setting runtime options, see [AddressSanitizer Runtime Options](./asan-runtime.md#runtime-options)"